hack/lib/cleanup.sh

```diff
@@ -94,8 +94,8 @@ function os::cleanup::tmpdir() {
         done
     fi
 
-		# delete any sub directory underneath BASETMPDIR
-    for directory in $( find "${BASETMPDIR}" -d 2 ); do
+    # delete any sub directory underneath BASETMPDIR
+    for directory in $( find "${BASETMPDIR}" -mindepth 2 -maxdepth 2 ); do
```

Comment:
I'll remove this commit later, but I need to have this change to be sure that fork_ami job is passing.